tsconfig.json

```diff
@@ -1,15 +1,19 @@
 {
   "compilerOptions": {
-    "target": "es5",
-    "module": "commonjs",
+    "target": "ES2020",
```

Comment:
What do the options in this file do in practice?

Reply:
The answer is probably not. Looking at this table the tl;dr is we don't support IE11, which is a pretty common thing to not support these days. https://compat-table.github.io/compat-table/es2016plus/. 

I can change it back to ES5 though, or ES6 which is what's used by our 1 other TS dependency - `goal-seek`.

More info: https://stackoverflow.com/questions/61835971/es7-es8-es9-es10-es11-browser-support